visual-testing-app/webpack.config.js

```diff
@@ -25,9 +25,7 @@ if (!info) {
 
 module.exports = {
   target: 'web',
-  // note: we have to use 'development' for now, because when we use production
-  // our emotion styles aren't displayed in our percy snapshots.
-  mode: 'development',
+  mode: 'production',
```

Comment:
Is this working now?

Reply:
I'm still working on it. I'm removing this change from this PR